### ISSUE TITLE
imgui example: Cannot use menus (does not recognize mouse input)

### ISSUE THREAD

**Issue by kyamant** [SaschaWillems/Vulkan#902]
The menus are not clickable nor draggable. Mouse can only rotate the vulkan logo image.
I verified against the version in proper_sync_dynamic_cb branch and that one works (albeit write-after-write hazard validation warnings).

**Comment by SaschaWillems** [comment]
Yeah, I never actually added that on the main branch. As you noticed, this will be fixed with the reworked sync branch, though I don't have a timeframe for that.

**Comment by kyamant** [comment]
I am somewhat confused by your comment. 
I am seeing this on the _main branch_ and it is having the problem I described. 
The one on the _sync branch_ is working. 
I recall this example working from sometime ago.

**Comment by SaschaWillems** [comment]
I'm not sure on this. I haven't done anything on that sample on master for some time. I can't rule out though, that I unintentionally broke something.

**Comment by kyamant** [comment]
I was able to run the example on my older machine (Vulkan SDK 1.2.162.1). So I copied main.cpp, external/imgui folder and glsl shaders to my main machine (Vulkan SDK 1.2.189.2) still the same problem. One thing I am noticing is when I close the window the example crashes on ` if (g.IO.Fonts && g.FontAtlasOwnedByContext)` of `void ImGui::Shutdown(ImGuiContext* context)` in the `imgui.cpp` file. I doubt the Vulkan SDK be problem since it seems fine with the version from proper_sync_dynamic_cb branch.

**Comment by kyamant** [comment]
Eureka!!!
After many experiments, it boiled down to the change in the default value of `VulkanExampleBase::settings.overlay` from false to true. 
Possible fix: in _main.cpp_ of _imgui_ example, set `settings.overlay = false` in the constructor.
